<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowClass</class>
 <widget class="QMainWindow" name="WindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="bacteriums_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_bacterium_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>add bacterium</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>340</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLineEdit" name="name_line_edit"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="species_label">
       <property name="text">
        <string>species</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="species_line_edit"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="size_line_edit"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="diseases_label">
       <property name="text">
        <string>diseases</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="diseases_lne_edit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="name_lable">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="size_lable">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="combo_box">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Window.qrc"/>
 </resources>
 <connections/>
</ui>
